sdk/communication/Azure.Communication.CallingServer/src/Models/PlaySource.cs

```diff
@@ -5,22 +5,11 @@
 using System.Collections.Generic;
 using System.Text;
 
-namespace Azure.Communication.CallingServer.Models
+namespace Azure.Communication.CallingServer
 {
     /// <summary> The PlaySource. </summary>
-    public class PlaySource
+    public abstract class PlaySource
```

Comment:
This code seems to have already been updated in master branch, such that it's not an abstract class anymore @arupdutta-msft